reactNativeProject/src/screens/Main/components/Title/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import { Text, View } from "react-native";
+import styles from "./styles";
+
+export default class Footer extends Component {
```

Comment:
Move export default to the end of the file. 
Since this component is stateless, it should be defined as a function.
Remember, a component should be defined as a class only when:
- It has a local state
- You need to define local methods
- You need the lifecycle methods